r/R/dplyr.R

```diff
@@ -390,6 +414,16 @@ collect.arrow_dplyr_query <- function(x, as_data_frame = TRUE, ...) {
         tab <- RecordBatch$create(!!!cols)
       }
     }
+    # Arrange rows
+    if (length(x$arrange_vars) > 0) {
+      x$arrange_vars <- get_field_names(x$arrange_vars)
```

Comment:
I don't understand how this works. I don't see how `get_field_names` will return something useful if the arrange_vars are expressions. And then on the next line you take `names()` of that anyway, so you're ignoring the expressions.

Reply:
Good catch; you're correct—`get_field_names()` serves no purpose here. It had a purpose earlier, but then I changed how this all works and forgot to remove it. It's removed in f8dd5b922706c9a0b54d0f43094253b5d0bf3e25.